up.go

```diff
@@ -75,13 +75,32 @@ var (
 	unsafeMode   = pflag.Bool("unsafe-full-throttle", false, "enable mode in which command is executed immediately after any change")
 	outputScript = pflag.StringP("output-script", "o", "", "save the command to specified `file` if Ctrl-X is pressed (default: up<N>.sh)")
 	debugMode    = pflag.Bool("debug", false, "debug mode")
+	versionFlag = pflag.Bool("version", false, "display version and credits")
 )
 
 func main() {
 	// Handle command-line flags
 	pflag.Parse()
 
 	log.SetOutput(ioutil.Discard)
+	if *versionFlag {
+		os.Stdout.WriteString("up: Ultimate Plumber v" + version + " https://github.com/akavel/up\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: Dependencies:\n")
+		os.Stdout.WriteString("up: \tTcell\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/gdamore/tcell\n")
+		os.Stdout.WriteString("up: \t\tLicense: Apache-2.0, ECL-2.0\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: \tgo-isatty\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/mattn/go-isatty\n")
+		os.Stdout.WriteString("up: \t\tLicense: MIT, MIT-feh, JSON, Xnet\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: \tpflag\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/spf13/pflag\n")
```

Comment:
1. Please note those are not all licenses that we're using yet. If you look into the [go.mod](https://github.com/akavel/up/blob/804be3cfb8982f83e850f28bef5730672b6d04da/go.mod) file, you can see there are a bit more (some of them are  transitive dependencies). Please try using `go list` to fetch all of them (see below).
 2. Please also add the `versions.sh` script that you used to run go-license-detector (rename it to `licenses.sh`). I believe it should be possible for us to add some smart `go list -f` call in the `versions.sh` script, to get it to automatically list all disk paths to all dependencies we're using, including the transitive ones<sup>†</sup>. Ideally, I would expect the final `licenses.sh` to look more or less like shown below:

        go list -f '{{ range .Deps }}{{ .Dir | println }}{{ end }}' | xargs -n1 license-detector

    (not tested, so not sure if this works as-is; please explore! :heart: )

 3. Also, please add a link to the `go-license-detector` project as a comment in the `licenses.sh` script.

———
<sup><sup>&dagger;</sup> I usually run `go help list` to find out how to build the proper incantation. There seems to be `type Package` described there, which contains fields `Deps` and `Dir` fields that may be useful to us. If that doesn't work, the `-m` flag may be helpful, with `type Module` containing `Dir` field. </sup>